test/run_test.py

```diff
@@ -217,7 +219,6 @@ def skip_test_p(name: str) -> bool:
     "distributed/_shard/sharded_tensor/ops/test_matrix_ops",
     "distributed/_shard/sharded_tensor/ops/test_softmax",
     "distributed/_shard/sharded_tensor/ops/test_tensor_ops",
-    "distributed/_shard/sharding_spec/test_sharding_spec",
```

Comment:
why we are removing sharding spec test from the skip list?

Reply:
Its duplicated, you can find it on line 204 already.